### ISSUE TITLE
Stop the training and execute the next lines of code

### ISSUE THREAD

**Issue by Rhuax** [fchollet/keras#2161]
Wouldn't be good to decide when to stop the execution of the fit() method during a training and execute the next lines? Or it would be bad in terms of performance since it needs to check every time if a specific key is pressed?
I think it would be good to stop training when a reasonable good loss value is seen.

**Comment by NasenSpray** [comment]
You can use callbacks to stop training at any point, e.g. see [EarlyStopping](http://keras.io/callbacks/#earlystopping)
